uberfire-project/uberfire-project-client/src/main/java/org/guvnor/common/services/project/client/context/WorkspaceProjectContext.java

```diff
@@ -67,19 +67,28 @@ public WorkspaceProjectContext(final Event<WorkspaceProjectContextChangeEvent> c
     }
 
     public void onOrganizationalUnitUpdated(@Observes final UpdatedOrganizationalUnitEvent event) {
-        contextChangeEvent.fire(new WorkspaceProjectContextChangeEvent(event.getOrganizationalUnit()));
+        WorkspaceProject updatedWorkspaceProject = new WorkspaceProject(event.getOrganizationalUnit(),
+                                                                        activeWorkspaceProject.getRepository(),
+                                                                        activeWorkspaceProject.getBranch(),
+                                                                        activeWorkspaceProject.getMainModule());
+        contextChangeEvent.fire(new WorkspaceProjectContextChangeEvent(updatedWorkspaceProject,
```

Comment:
@Rikkola I still have to add the other active modules and packages, but I took you advice and I've created the new WorkspaceProject